chef/cookbooks/openstack-database/templates/default/trove-taskmanager.conf.erb

```diff
@@ -85,11 +134,12 @@ dns_instance_entry_factory = trove.dns.designate.driver.DesignateInstanceEntryFa
 dns_endpoint_url = http://127.0.0.1/v1/
 dns_service_type = dns
 
+# Neutron
+network_driver = trove.network.nova.NovaNetwork
+default_neutron_networks =
```

Comment:
No, according documentation and configuration sample for taskamanager, it is fine